Arrowgene.Ddon.GameServer/Characters/ItemManager.cs

```diff
@@ -291,7 +291,7 @@ public List<CDataItemUpdateResult> AddItem(DdonServer<GameClient> server, Charac
                 else
                 {
                     // Move to storage box without stack limit if it's not equipment
-                    return DoAddItem(server.Database, character, destinationStorage, itemId, num, connectionIn:connectionIn);
+                    return DoAddItem(server.Database, character, destinationStorage, itemId, num, STACK_BOX_MAX, connectionIn: connectionIn);
```

Comment:
Be consistent with the space in the `connectionIn:connectionIn`. I noticed you added a space in this line but not in the previous one.